dotcom-rendering/.storybook/decorators/splitThemeDecorators.tsx

```diff
@@ -55,27 +60,63 @@ export const splitTheme =
 			<>
 				<div css={[orientation === 'horizontal' ? rowCss : columnCss]}>
 					<div
-						className="left-lightTheme"
 						css={[
-							splitCss,
-							lightStoryCss,
+							splitCss(),
+							lightStoryCss(),
 							css(paletteDeclarations(format, 'light')),
 						]}
+						style={{ ['--column-width' as any]: 'auto', ['--light-mode-float' as any]: 'none' }}
 					>
 						<div css={headerCss}>Light Theme ☀️</div>
 						<Story />
 					</div>
 					<div
-						className="right-darkTheme"
 						css={[
-							splitCss,
-							darkStoryCss,
+							splitCss(),
+							darkStoryCss(),
 							css(paletteDeclarations(format, 'dark')),
 						]}
+						style={{ ['--column-width' as any]: 'auto',  ['--dark-mode-float' as any]: 'none'}}
```

Comment:
Not a fan of `as any`, but we can address in a follow-up PR.

I think it would be preferable to add this to our `index.d.ts`:

```ts
declare namespace React {
	interface CSSProperties {
		[key: `--${string}`]: string;
	}
}
```

Reply:
Addressed in #9413